### DISCUSSION TITLE
Table of Contents file naming convention

### DISCUSSION THREAD

**Discussion by PricedTransparently** [CMSgov/price-transparency-guide#372] [Q&A] [answered]
The current naming convention for the table of contents file calls for putting the payer name in the file name. Since this table of contents file can index many employer groups, it is possible that different reporting plans in the table of contents could have different payers. In this scenario, how would we be able to adhere to the file's naming convention? 

Is there something we could substitute the payer name with? Perhaps the entity name?

**Comment by dolinsky** [comment, marked as answer]
Table of Contents should really be thought of as being a Payer Table of Contents, so there would be as many ToC published on a monthly basis as there are payers.

**Reply by RLTx1391** [reply]
I do agree with @PricedTransparently's suggestion on entity or other name. If there are multiple employers that would gave the same MRF, it seems unnecessarily duplicative to need to create a separate ToC for each one.
